apps/tui/init_app.py

```
@@ -114,17 +135,54 @@ async def on_mount(self) -> None:
+            configured = bool(await self._conn.fetchval("SELECT is_agent_configured()"))
+            if configured:
+                raw = await self._conn.fetchval("SELECT get_init_profile()")
+                profile = json.loads(raw) if isinstance(raw, str) else (raw or {})
+                llm = await self._conn.fetchval("SELECT get_config('llm.chat')")
+                llm = json.loads(llm) if isinstance(llm, str) else (llm or {})
+                summary = {
+                    "name": (profile.get("agent", {}) or {}).get("name", "your agent"),
+                    "provider": llm.get("provider", "?"),
+                    "model": llm.get("model", "?"),
+                }
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)